tests/by-util/test_seq.rs

```diff
@@ -698,7 +698,7 @@ fn test_parse_error_hex() {
     new_ucmd!()
         .arg("0xlmnop")
         .fails()
-        .usage_error("invalid hexadecimal argument: '0xlmnop'");
+        .usage_error("invalid floating point argument: '0xlmnop'");
 }
 
 #[test]
```

Comment:
please add more tests 
the unit tests are nice but integration tests are much better :)

Reply:
No problem